### ISSUE TITLE
why config.getini in only one place

### ISSUE THREAD

**Issue by birdsarah** [davehunt/pytest-selenium#31]
Am curious about https://github.com/davehunt/pytest-selenium/blob/master/pytest_selenium/pytest_selenium.py#L46 

It reads:
```python
url = config.option.base_url or config.getini('selenium_base_url')
```

1) Why is this the only place that `getini` is available for configuration - shouldn't it be all or nothing for consistency.
2) Why is it `selenium_base_url` and not `base_url`?

**Comment by davehunt** [comment]
It's not the only place, the cloud provider configuration can also be set in the ini file. The reason why only these are set is that these seemed reasonable to have project/system defaults. Each project would likely have its own default base URL, and the cloud credentials would likley be system defaults. I did consider adding more, but these for now seemed the most appropriate.

As for the naming, I agree that `base_url` makes more sense and I'll rename it. The reason for this is that I felt it was worth grouping the options by name, as they share space with the other pytest configuration options, but I've made base URL its own fixture that the selenium fixture happens to depend on so the rename makes sense.

I'll close this now, and go ahead with the rename.
